docs/v2/spec/2018_Public_TS_Schema.csv

```diff
@@ -0,0 +1,13 @@
+﻿Field Name,Field Data Type,Valid Values,Descriptions and Examples

```

Comment:
Please change "descriptions and examples" to "notes". 

If descriptions or examples are needed users can refer to the larger file. I'm seeing a notes column as explaining odd edge cases (of which I can't think of one for the TS).

Please also remove the examples. The names of the fields along with the data types are sufficient in this context to explain what should be present in the file.

Reply:
Sure - I can change this.